blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/modules/scan/ScanModuleConfig.java

```diff
@@ -96,6 +137,8 @@ public void validate(PropertyGroupReport propertyGroupReport) {
         validateList(propertyGroupReport, ScanModuleProperty.REPOS, repos,
             String.format("No valid repositories specified. Please set the %s or %s property with valid repositories", ScanModuleProperty.REPOS.getKey(), ScanModuleProperty.REPOS_CSV_PATH.getKey()));
         validateBoolean(propertyGroupReport, ScanModuleProperty.METADATA_BLOCK, metadataBlockEnabled);
+        validateBoolean(propertyGroupReport, ScanModuleProperty.POLICY_BLOCK, policyBlockedEnabled);
+        validateList(propertyGroupReport, ScanModuleProperty.POLICY_SEVERITY_TYPES, policySeverityTypes, "No severity types were provided to block on.");
```

Comment:
Do you care about validating whether or not the Policy Repos specified exist?

Reply:
I originally thought no because If the field is left blank, that means it should inherit all of the repositories for that module. If the plugin inherits an empty list, then the validation would fail because the module needs at least one repo to be enabled. I will add a validation anyway in case that changes in the future.